cluster-autoscaler/cloudprovider/azure/azure_config.go

```diff
@@ -59,6 +59,9 @@ const (
 	// auth methods
 	authMethodPrincipal = "principal"
 	authMethodCLI       = "cli"
+
+	// toggle
+	dynamicInstanceListDefault = false
```

Comment:
you can get rid of this and set to false directly.

Reply:
like the `EnableDynamicInstanceList` boolean will always default to false if unspecified so no need.

Reply:
I just saw the other fields do that but it feels odd lol

Reply:
Yes, that's why I followed this pattern. In that case, I won't make any changes here?